plugins/ILibPlugin/index.js

```diff
@@ -183,17 +185,10 @@ class ILibPlugin {
 		if (typeof this.options.resources === 'undefined') {
 			this.options.resources = 'resources';
 		}
-		this.options.bundles = this.options.bundles || {};
-		if (typeof this.options.bundles.moonstone === 'undefined') {
-			if (pkgName === '@enact/moonstone') {
-				this.options.bundles.moonstone = 'resources';
-				this.options.resources = '_resources_';
-			} else {
-				const moonstone = packageSearch(process.cwd(), path.join('@enact', 'moonstone'));
-				if (moonstone) {
-					this.options.bundles.moonstone = path.join(moonstone, 'resources');
-				}
-			}
+		if ((!this.options.bundles || !this.options.bundles.moonstone) && pkgName === '@enact/moonstone') {
+			this.options.bundles = this.options.bundles || {};
+			this.options.bundles.moonstone = 'resources';
```

Comment:
Would this logic be automatically handled by the new "autodetect" logic below? I'm not sure how best to test this myself.

Reply:
This specific logic is for building within `@enact/moonstone` itself, not moonstone as a 'theme' per say. This is mainly a relic of the past (where we had Karma building the test files within `@enact/moonstone` itself).  It ideally would not be encounted nowadays, however I'm hesitant to remove specific-case handling just because we aren't directly using it right now especially when it doesn't affect overall functionality.

Reply:
mmm ... unit tests ... ;)